eks-anywhere-baremetal/Addons/Partner/newrelic/newrelic.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: newrelic
+  namespace: newrelic
+spec:
+  chart:
+    spec:
+      chart: nri-bundle
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: newrelic-charts
+        namespace: flux-system
+      version: 5.0.4
```

Comment:
The current nri-bundle release is [5.0.16](https://github.com/newrelic/helm-charts/blob/9266691a4051a1e1b02ac733e5a2926b7fc7cfc8/charts/nri-bundle/Chart.yaml#LL19C16-L19C16)